javascript/i18n/phonenumbers/phonenumberutil.js

```diff
@@ -778,11 +778,11 @@ i18n.phonenumbers.PhoneNumberUtil.CAPTURING_EXTN_DIGITS_ =
 i18n.phonenumbers.PhoneNumberUtil.EXTN_PATTERNS_FOR_PARSING_ =
     i18n.phonenumbers.PhoneNumberUtil.RFC3966_EXTN_PREFIX_ +
     i18n.phonenumbers.PhoneNumberUtil.CAPTURING_EXTN_DIGITS_ + '|' +
-    '[ \u00A0\\t,]*' +
+    '[ \xA0\\t,]*' +
     '(?:e?xt(?:ensi(?:o\u0301?|\u00F3))?n?|\uFF45?\uFF58\uFF54\uFF4E?|' +
-    'доб|\u0434\u043E\u0431|ДОБ|' +
+    'доб|ДОБ|' + // '\u0434\u043E\u0431|\u0414\u041E\u0411'
```

Comment:
Thanks so much for the info Mathias.
Then this might be the reason why we are using only ASCII-safe characters in source code and not using them directly доб. Sounds good to me make the changes in the same patch. I can test it for you if needed.

Reply:
Pushed the change I suggested.